conf/country-code.json

```diff
@@ -1,998 +0,0 @@
-  },
-  {
-    "countryCode": "MG",
-    "country": "Madagascar"
-  },
-  {
-    "countryCode": "MW",
-    "country": "Malawi"
-  },
-  {
-    "countryCode": "MY",
-    "country": "Malaysia"
-  },
-  {
-    "countryCode": "MV",
-    "country": "Maldives"
-  },
-  {
-    "countryCode": "ML",
-    "country": "Mali"
-  },
-  {
-    "countryCode": "MT",
-    "country": "Malta"
-  },
-  {
-    "countryCode": "MH",
-    "country": "Marshall Islands (the)"
-  },
-  {
-    "countryCode": "MQ",
-    "country": "Martinique"
-  },
-  {
-    "countryCode": "MR",
-    "country": "Mauritania"
-  },
-  {
-    "countryCode": "MU",
-    "country": "Mauritius"
-  },
-  {
-    "countryCode": "YT",
-    "country": "Mayotte"
-  },
-  {
-    "countryCode": "MX",
-    "country": "Mexico"
-  },
-  {
-    "countryCode": "FM",
-    "country": "Micronesia (Federated States of)"
-  },
-  {
-    "countryCode": "MD",
-    "country": "Moldova (the Republic of)"
-  },
-  {
-    "countryCode": "MC",
-    "country": "Monaco"
-  },
-  {
-    "countryCode": "MN",
-    "country": "Mongolia"
-  },
-  {
-    "countryCode": "ME",
-    "country": "Montenegro"
-  },
-  {
-    "countryCode": "MS",
-    "country": "Montserrat"
-  },
-  {
-    "countryCode": "MA",
-    "country": "Morocco"
-  },
-  {
-    "countryCode": "MZ",
-    "country": "Mozambique"
-  },
-  {
-    "countryCode": "MM",
-    "country": "Myanmar"
-  },
-  {
-    "countryCode": "NA",
-    "country": "Namibia"
-  },
-  {
-    "countryCode": "NR",
-    "country": "Nauru"
-  },
-  {
-    "countryCode": "NP",
-    "country": "Nepal"
-  },
-  {
-    "countryCode": "NL",
-    "country": "Netherlands (the)"
-  },
-  {
-    "countryCode": "NC",
-    "country": "New Caledonia"
-  },
-  {
-    "countryCode": "NZ",
-    "country": "New Zealand"
-  },
-  {
-    "countryCode": "NI",
-    "country": "Nicaragua"
-  },
-  {
-    "countryCode": "NE",
-    "country": "Niger (the)"
-  },
-  {
-    "countryCode": "NG",
-    "country": "Nigeria"
-  },
-  {
-    "countryCode": "NU",
-    "country": "Niue"
-  },
-  {
-    "countryCode": "NF",
-    "country": "Norfolk Island"
-  },
-  {
-    "countryCode": "MK",
-    "country": "North Macedonia"
-  },
-  {
-    "countryCode": "MP",
-    "country": "Northern Mariana Islands (the)"
-  },
-  {
-    "countryCode": "NO",
-    "country": "Norway"
-  },
-  {
-    "countryCode": "OM",
-    "country": "Oman"
-  },
-  {
-    "countryCode": "PK",
-    "country": "Pakistan"
-  },
-  {
-    "countryCode": "PW",
-    "country": "Palau"
-  },
-  {
-    "countryCode": "PS",
-    "country": "Palestine, State of"
-  },
-  {
-    "countryCode": "PA",
-    "country": "Panama"
-  },
-  {
-    "countryCode": "PG",
-    "country": "Papua New Guinea"
-  },
-  {
-    "countryCode": "PY",
-    "country": "Paraguay"
-  },
-  {
-    "countryCode": "PE",
-    "country": "Peru"
-  },
-  {
-    "countryCode": "PH",
-    "country": "Philippines (the)"
-  },
-  {
-    "countryCode": "PN",
-    "country": "Pitcairn"
```

Comment:
E.g. not in location-autocomplete-canonical-list.json

Reply:
country-code shouldn't be considered